apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/fix_sporadic_bloom_filter_jdbc_errors.sql

```diff
@@ -0,0 +1,10 @@
+/* 
+ BFD-1505
+ There are periodic JDBC errors related to the Loaded Filter Manager bloom filter implementation.
+ 
+ This corrects that issue by eliminating the sequence scan on loaded_batches with an index.
```

Comment:
Not sure this is true; it MAY fix the problem. What it does do however, is to (hopefully) encourage the Postgres engine to not do a sequential scan. So maybe, the comment could be something to that effect.

Reply:
Fixing